apl-core/src/test/java/com/apollocurrency/aplwallet/apl/core/account/service/AliasServiceImplTest.java

```diff
@@ -0,0 +1,341 @@
+package com.apollocurrency.aplwallet.apl.core.account.service;
+
+import com.apollocurrency.aplwallet.apl.core.account.dao.AliasOfferTable;
+import com.apollocurrency.aplwallet.apl.core.account.dao.AliasTable;
+import com.apollocurrency.aplwallet.apl.core.app.Alias;
+import com.apollocurrency.aplwallet.apl.core.app.AliasOffer;
+import com.apollocurrency.aplwallet.apl.core.app.Blockchain;
+import com.apollocurrency.aplwallet.apl.core.app.Transaction;
+import com.apollocurrency.aplwallet.apl.core.db.DbClause;
+import com.apollocurrency.aplwallet.apl.core.transaction.messages.MessagingAliasAssignment;
+import com.apollocurrency.aplwallet.apl.core.transaction.messages.MessagingAliasSell;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Spy;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+/**
+ * @author silaev-firstbridge on 3/19/2020
+ */
+@ExtendWith(MockitoExtension.class)
+class AliasServiceImplTest {
+    @Mock
+    private AliasTable aliasTable;
+    @Mock
+    private AliasOfferTable offerTable;
+    @Mock
+    private Blockchain blockchain;
+    @InjectMocks
+    @Spy
+    private AliasServiceImpl aliasService;
```

Comment:
Need to avoid partial mocking (spies) since it show that code was designed not so good as needed

Reply:
Generally, it's try, but not here. Spy helps to avoid testing    AliasServiceImpl=> changeOwner(long newOwnerId, String aliasName) twice